### PR TITLE
Basic Shades

### PR DESCRIPTION
**Pull request by lizardqueenlexi**
## About The Pull Request

Makes shades into basic mobs. As they are solely player-controller and have no AI, this was a very simple conversion.

Things of note:
- I've made shades use the same "theme" system as constructs, to determine their drops and coloration - as opposed to these things being manually set by the type of soulstone they're held in.
- I've reorganized files slightly, putting both constructs and shades in a new "cult" basic mob folder.

That's more or less it. As I said, shades are simple.
## Why It's Good For The Game

Basic-izes another mob and cleans up the code a little. Removes the last cult-related simplemob, too.
## Changelog
:cl:
refactor: Shades now use the basic mob framework. Please report any bugs.
/:cl:

### CODE REVIEW

File: code/modules/mob/living/basic/cult/shade.dm [hunk 1]
```diff
@@ -10,45 +10,59 @@
...
 
-/mob/living/simple_animal/shade/Initialize(mapload)
+/mob/living/basic/shade/Initialize(mapload)
 	. = ..()
 	AddElement(/datum/element/simple_flying)
+	var/list/remains = string_list(remains_by_theme[theme])
+	if(length(remains))
+		AddElement(/datum/element/death_drops, remains)
 	add_traits(list(TRAIT_HEALS_FROM_CULT_PYLONS, TRAIT_SPACEWALK, TRAIT_VENTCRAWLER_ALWAYS), INNATE_TRAIT)
+	if(!isnull(theme))
```
Comment (Jacquerel) [inline code/modules/mob/living/basic/cult/shade.dm:46]: If theme can be null here then wouldn't `remains_by_theme[theme]` earlier runtime?

**Review by Jacquerel — CHANGES_REQUESTED** [review]
Not anything you should change for this PR which is a good pure refactor, but it's always been weird to me that shades are so boring. 
They're ghosts? But ones which are stopped by walls and can... ventcrawl. I guess they're made of some kind of evil gas.

**Comment by lizardqueenlexi** [comment]
> Not anything you should change for this PR which is a good pure refactor, but it's always been weird to me that shades are so boring. They're ghosts? But ones which are stopped by walls and can... ventcrawl. I guess they're made of some kind of evil gas.

Literally the only thing they have going for them is that they're extremely fast, but I'm really not sure what they're _for_. Being annoying?

**Comment by Jacquerel** [comment]
Resolve conflicts and hopefully that won't disable automerge

**Comment by san7890** [comment]
it will disable auto-merge because any push from someone who doesn't have write access will turn it off. it worked the other day because I have write access

**Pull request merged**